data/json/mutations/mutations.json

```diff
@@ -3598,6 +3598,28 @@
       "strength_damage": { "damage_type": "cut", "amount": 4 }
     }
   },
+  {
+    "type": "mutation",
+    "id": "TOE_CLAWS",
+    "name": { "str": "Clawed Feet" },
+    "copy-from": "RAP_TALONS",
+    "description": "Your toenails have grown long and sharp, preventing you from wearing shoes, but providing you with a set of powerful claws on your feet.  Fortunately they don't impede your movement.",
+    "category": [ "LIZARD" ]
+  },
+  {
+    "type": "mutation",
+    "id": "FOOT_TALONS",
+    "name": { "str": "Taloned Feet" },
+    "copy-from": "RAP_TALONS",
+    "description": "Your feet have shifted into bird-like talons.  Instead of toes you now have three long talons at the front and one at the back, ending in wicked claws and preventing you from wearing shoes.  Fortunately they don't impede your movement.",
+    "category": [ "BIRD" ],
+    "attacks": {
+      "attack_text_u": "You slash %s with a talon",
```

Comment:
Do they slash with a single talon?

Reply:
Nope, somehow this change didn't make it into the PR. Working on it.

Reply:
Cool thanks :)